build/extract-l10n.js

```diff
@@ -0,0 +1,25 @@
+const { GettextExtractor, JsExtractors } = require('gettext-extractor');
+
+let extractor = new GettextExtractor();
+
+extractor
+    .createJsParser([
+        JsExtractors.callExpression('t', {
+            arguments: {
+                text: 0,
+                context: 1
+            }
+        }),
+        JsExtractors.callExpression('n', {
+            arguments: {
+                text: 1,
+                textPlural: 2,
+                context: 3
```

Comment:
Actually count js 3 and context is 4

Reply:
Yep, count is definitely 3. I think we don't even use context. At least not for `ngettext` https://github.com/alexanderwallin/node-gettext#Gettext